.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "circleci.persistedProjectSelection": [
+        "gh/Remi-Gau/bids-examples"
+    ]
+}
```

Comment:
Maybe let's add `.vscode` to `.gitignore`? I don't think we want this in the repo.

Reply:
oh dear. 
"git add --all" strikes again